src/apis/openai.ts

```diff
@@ -71,23 +71,27 @@ const elysia = new Elysia({
           },
         };
       }
-      const streamResult = [
-        {
-          id: `${Date.now()}`,
-          object: "chat.completion.chunk",
-          created: Date.now(),
-          model: "meowgpt",
-          system_fingerprint: "fp_44709d6fcb",
-          choices: [
-            {
-              index: 0,
-              delta: { role: "assistant", content: "" },
-              logprobs: null,
-              finish_reason: null,
-            },
-          ],
-        },
-        {
+
+      const chunkMeowMessages = meowMessage.split("");
+
+      yield {
+        id: `${Date.now()}`,
+        object: "chat.completion.chunk",
+        created: Date.now(),
+        model: "meowgpt",
+        system_fingerprint: "fp_44709d6fcb",
+        choices: [
+          {
+            index: 0,
+            delta: { role: "assistant", content: "" },
+            logprobs: null,
+            finish_reason: null,
+          },
+        ],
+      };
```

Comment:
This seems to make Elysia output the result as raw JSON. However, OpenAI clients expect the result to be delivered as [server sent event format](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events#event_stream_format).

To resolve this, I have to convert the output object into a string that conforms to that server sent event format.